.gitignore

```diff
@@ -2,6 +2,8 @@
 *.pico
 *.dSYM
 *.exe
+*.dll
+*.obj
```

Comment:
What is this _pthreadVC2.dll_ of which you speak? Why would it appear in the htslib source directory rather than some compiler directory? (If there's some larger pull request you're working towards, showing the draft end product would be illuminating…)